.github/workflows/deploy.yml

```
@@ -47,20 +47,22 @@ jobs:
 
       # https://github.com/pypa/cibuildwheel
       - name: Build wheels
-        uses: pypa/cibuildwheel@v3.1.4
+        uses: pypa/cibuildwheel@v3.2.0
         with:
           output-dir: dist
         # Options are supplied via environment variables:
         env:
           # Build wheels for the currently selected architecture.
           CIBW_ARCHS: ${{ matrix.cibw_arch }}
           # Include latest Python beta.
-          CIBW_ENABLE: cpython-prerelease pypy
-          # Skip EOL Python versions.
-          CIBW_SKIP: "pp310*"
+          CIBW_ENABLE: cpython-prerelease pypy graalpy
+          # Skip EOL Python versions
+          CIBW_SKIP: "pp310* gp311*"
```

In cibuildwheel 3.2: "PyPy 3.10 was moved to pypy-eol in the enable option, as it is now end-of-life" 
```suggestion
          CIBW_SKIP: "gp311*"
```